packages/ui-table/src/Table/README.md

```diff
@@ -167,141 +351,412 @@ class Example extends React.Component {
+            large: { layout: 'auto' }
+          }}
+        >
+          {(props) => (
+            <div>
+              {props.layout !== 'stacked' && (
+                <View display="block" margin="0 0 medium">
+                  {this.renderOptions()}
+                </View>
+              )}
+
+              <Table
+                caption={`${caption}: sorted by ${sortBy} in ${direction} order`}
+                {...props}
+              >
+                <Table.Head renderSortLabel="Sort by">
+                  {this.renderHeaderRow(direction)}
+                </Table.Head>
+                <Table.Body>
+                  {sortedRows.map((row) => (
+                    <Table.Row key={row.id}>
+                      {headers.map(({ id, renderCell }) => (
+                        <Table.Cell key={id} textAlign={colTextAligns[id]}>
+                          {renderCell ? renderCell(row[id]) : row[id]}
+                        </Table.Cell>
+                      ))}
+                    </Table.Row>
+                  ))}
+                </Table.Body>
+              </Table>
+              <Alert
+                liveRegion={() => document.getElementById('flash-messages')}
+                liveRegionPoliteness="polite"
+                screenReaderOnly
+              >
+                {`Sorted by ${sortBy} in ${direction} order`}
+              </Alert>
+            </div>
+          )}
+        </Responsive>
+      )
+    }
+  }
+
+  render(
+    <SortableTable
+      caption="Top rated movies"
+      headers={[
+        {
+          id: 'rank',
+          text: 'Rank',
+          width: '15%'
+        },
+        {
+          id: 'title',
+          text: 'Title',
+          width: '55%'
+        },
+        {
+          id: 'year',
+          text: 'Year',
+          width: '15%'
+        },
+        {
+          id: 'rating',
+          text: 'Rating',
+          width: '15%',
+          renderCell: (rating) => rating.toFixed(1)
+        }
+      ]}
+      rows={[
+        {
+          id: '1',
+          rank: 1,
+          title: 'The Shawshank Redemption',
+          year: 1994,
+          rating: 9.3
+        },
+        {
+          id: '2',
+          rank: 2,
+          title: 'The Godfather',
+          year: 1972,
+          rating: 9.2
+        },
+        {
+          id: '3',
+          rank: 3,
+          title: 'The Godfather: Part II',
+          year: 1974,
+          rating: 9.0
+        },
+        {
+          id: '4',
+          rank: 4,
+          title: 'The Dark Knight',
+          year: 2008,
+          rating: 9.0
+        },
+        {
+          id: '5',
+          rank: 5,
+          title: '12 Angry Men',
+          year: 1957,
+          rating: 8.9
+        }
+      ]}
+    />
+  )
+  ```
 
+- ```javascript
+  const SortableTable = ({ caption, headers, rows }) => {
     const initialColWidth = {}
     headers.forEach((header) => {
       initialColWidth[header.id] = 'start'
     })
```

Comment:
This whole functional example doesn't work, it throws an error: TypeError: Cannot read properties of undefined (reading 'onRequestSort'). Can you please look into this again?

Reply:
I couldn't reproduce this issue in my local. I can sort the table as expected. Could you provide more context like if there is any like special settings set etc.?

Reply:
I checked it again, it works now 👍